src/assets/assets.service.ts

```diff
@@ -63,7 +63,9 @@ export class AssetsService {
   ): Promise<Asset> {
     return prisma.asset.update({
       data: {
```

Comment:
This change is intentional? Seems like it is accomplishing the same thing as before

Reply:
Yeah it is intentional. I think there's a weird bug in the Prisma engine, so I'm trying this operation instead of addition on the TS side so the DB runner can deal with it